web/src/components/Header/Header.tsx

```diff
@@ -1,64 +1,33 @@
-import {QuestionCircleOutlined} from '@ant-design/icons';
 import {useTour} from '@reactour/tour';
+import {Typography} from 'antd';
 import Logo from 'assets/Logo.svg';
-import {DOCUMENTATION_URL, GITHUB_URL} from 'constants/Common.constants';
-import {FC} from 'react';
+import {FC, useState} from 'react';
 import {Link, useLocation} from 'react-router-dom';
-import HomeAnalyticsService from 'services/Analytics/HomeAnalytics.service';
+import HomeAnalyticsService from '../../services/Analytics/HomeAnalytics.service';
```

Comment:
I would prefer to keep the absolute path here.

Reply:
done